src/etc/inc/globals.inc

```diff
@@ -74,7 +74,7 @@ $g = array(
 	"disablecrashreporter" => false,
 	"crashreporterurl" => "https://crashreporter.pfsense.org/crash_reporter.php",
 	"debug" => false,
-	"latest_config" => "17.3",
+	"latest_config" => "17.4",
```

Comment:
This is out of date. Please check the current latest_config version and update accordingly.